.github/workflows/ci.yml

```diff
@@ -19,17 +19,41 @@ jobs:
 
     steps:
       - uses: actions/checkout@v3
+
       - name: Install Node
         uses: actions/setup-node@v3
         with:
           node-version: 14.x
-          cache: npm
+
+      - uses: pnpm/action-setup@v2
```

Comment:
WDYT about using https://github.com/NullVoxPopuli/action-setup-pnpm? Should reduce the bilerplate.

Reply:
Good idea! Will do 👌

Reply:
Done! Thanks for this suggestion, it's way better now!

Reply:
Updated floating dependencies config to use action setup with no-lockfile option:

```
      - uses: NullVoxPopuli/action-setup-pnpm@v1
        name: Install pnpm
        with:
          pnpm-version: 7.29.0
          no-lockfile: true
```